site/src/sphinx/advanced-retry.rst

```diff
@@ -1,8 +1,154 @@
+                ...
+                if (result.headers().status() == CONFLICT) {
+                    return Optional.of(backoff);
+                }
+                return Optional.empty();
+            });
+        }
+    };
+
+This will retry when the response's status is ``409``.
+
+.. note::
+
+    We declare a ``backoff`` as a member and return it. The inside of RetryingClient_ tracks the
+    reference of the ``backoff`` and increases the counter one by one. The counter will be used
+    as the ``numAttemptsSoFar`` to get the next delay in Backoff_. we will take a close look what
+    the Backoff_ is at the bottom of this page.
+
+You can return different ``backoffs`` according to the response.
+
+.. code-block:: java
+
+    new RetryStrategy<HttpRequest, HttpResponse>() {
+        final Backoff backoffOnServerError = RetryStrategy.defaultBackoff;
+        final Backoff backoffOnConflict = Backoff.fixed(100);
+
+        @Override
+        public CompletableFuture<Optional<Backoff>> shouldRetry(HttpRequest request,
+                                                                HttpResponse response) {
+            return response.aggregate().handle((result, cause) -> {
+                ...
+                if (result.headers().status() == SERVER_ERROR) {
+                    return Optional.of(backoffOnServerError);
+                } else if (result.headers().status() == CONFLICT) {
+                      return Optional.of(backoffOnConflict);
+                }
+                return Optional.empty();
+            });
+        }
+    };
+
+.. _backoff-section:
+
+Backoff_
+--------
+
+You can use a ``backoff`` to determine the delay between attempts. There are three ``backoffs`` in Armeria:
+
+- FixedBackoff_
+- RandomBackoff_
+- ExponentialBackoff_
+
+``FixedBackoff`` returns the fixed milliseconds which is specified when it is created using
+``Backoff.fixed(delay)``. ``RandomBackoff`` returns a random milliseconds between minimum and maximum delays.
+``ExponentialBackoff`` returns a milliseconds which is exponentially multiplied in each attempt.
+If you invoke ``withJitter(jitterRate)`` on the instance, the delay will have the jitter.
+For more information, please refer to the API documentation of the `com.linecorp.armeria.client.retry`_ package.
+
+RetryingClient with logging
+---------------------------
+
+You can use RetryingClient_ with LoggingClient_ to log. If you want to log the first request and the last
+response(no matter if it's successful or not), decorate RetryingClient_ with LoggingClient_. That is:
+
+.. code-block:: java
+
+    RetryStrategy strategy = RetryStrategy.onServerErrorStatus();
+    HttpClient client = new HttpClientBuilder(...)
+                              .decorator(RetryingHttpClient.newDecorator(strategy))
+                              .decorator(LoggingClient.newDecorator())
+                              .build();
+
+If you want to log all of the requests and responses, then do the reverse:
```

Comment:
- Could you also paste the example log messages for both cases so that a user knows the difference clearly?